src/redux/teachers/teachers.operations.ts

```diff
@@ -18,3 +22,22 @@ const getData = createAsyncThunk(
 )
 
 export { getData }
+
+const bookLesson = createAsyncThunk(
+  'teachers/bookLesson',
+  async (
+    { teacherId, lessonData }: { teacherId: number; lessonData: any },
+    { rejectWithValue }
+  ) => {
+    try {
+      const res = await axios.post(`${baseUrl}/book`, { teacherId, lessonData })
+      return res.data
+    } catch (err) {
+      if (err instanceof Error) {
+        return rejectWithValue(err.message)
+      }
+    }
+  }
+)
+
+export { bookLesson }
```

Comment:
we haven't method 'post', do it in reducers: {}, or if im not right, solve me it